include/flatbuffers/idl.h

```diff
@@ -245,6 +245,7 @@ struct FieldDef : public Definition {
   bool native_inline;  // Field will be defined inline (instead of as a pointer)
                        // for native tables if field is a struct.
   bool flexbuffer; // This field contains FlexBuffer data.
+  StructDef* nested_flatbuffer; // This field contains nested FlatBuffer data.
```

Comment:
Sorry for code-blindness hitting in when switching between coding styles (I'm using a modified 'Allman' in my codebase). Fixed.